.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### 1. Why is this change necessary?
+
+
+### 2. What does this change do, exactly?
+
+
+### 3. Describe each step to reproduce the issue or behaviour.
+
+
+### 4. Please link to the relevant issues (if any).
```

Comment:
Can be removed if covered by 1. https://github.com/shopware/SwagMigrationAssistant/pull/89/files#r2526092590